internal/router/router.go

```diff
@@ -36,11 +37,15 @@ func Register(svcCfg *config.SvcConfig) *mux.Router {
 }
 
 func attachHtmlPdfServiceRoutes(m *mux.Router, svcCfg *config.SvcConfig) *mux.Router {
-	dataSource := datasource.NewDummyDs(&svcCfg.DummySvc)
+	dataSource := datasource.NewRedisDs(&svcCfg.CacherSvc)
+	htmlTopdfSvc := htmlToPdf.NewWkHtmlToPdfSvc()
 
-	svc := handler.NewHtmlPdfService(dataSource)
+	svc := handler.NewHtmlPdfService(dataSource, htmlTopdfSvc)
 
 	m.HandleFunc("/ping", svc.Ping).Methods(http.MethodPost)
-
+	m.HandleFunc("/register", svc.Upload).Methods(http.MethodPost)
+	m.HandleFunc("/generate/{id}", svc.ConvertToPdf).Methods(http.MethodPost)
+	m.HandleFunc("/register/{id}", svc.ReplaceHtml).Methods(http.MethodPut)
+	m.HandleFunc("/health", svc.Health).Methods(http.MethodGet)
```

Comment:
Why is the health endpoint again added? The generic health endpoint already does this.

Reply:
Done